src/components/scores/concept-coach-cell.cjsx

```diff
@@ -10,15 +10,49 @@ ConceptCoachCell = React.createClass
 
   mixins: [CellStatusMixin] # prop validation
 
+  getInitialState: ->
+    placement: 'right'
+
+  setPopoverPosition: ->
```

Comment:
oh...we do have some existing code that tries to set popover position-- [TutorPopover](https://github.com/openstax/tutor-js/blob/master/src/components/tutor-popover.cjsx) component